internal/auth0/client/resource.go

```diff
@@ -818,6 +818,28 @@ func NewResource() *schema.Resource {
 								},
 							},
 						},
+						"office365": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							MaxItems:    1,
+							Description: "Microsoft Office 365 SSO configuration.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"domain": {
+										Description: "Your Office 365 domain name. e.g. `acme-org.com`.",
```

Comment:
```suggestion
										Description: "Your Office 365 domain name, for example `acme-org.com`.",
```
The docs team's guidelines instruct not to use latin abbreviations.